src/gui/qgssublayersdialog.h

```diff
@@ -59,10 +88,21 @@ class GUI_EXPORT QgsSublayersDialog : public QDialog, private Ui::QgsSublayersDi
     typedef QList<QgsSublayersDialog::LayerDefinition> LayerDefinitionList;
 
     //! Constructor for QgsSublayersDialog
+
+    /**
+     * Construct a new QgsSublayersDialog object - a dialog to select which sub layers to be imported from a file (e.g. from geopackage or zipfile)
+     *
+     * @param providerType provider type
```

Comment:
You need to use "\param" syntax instead of "@param", so that the sipify script can do its magic and create the correct Python docstring.